pkg/apis/application/v1alpha1/applicationset_types.go

```diff
@@ -524,6 +524,8 @@ type SCMProviderGeneratorBitbucketServer struct {
 	API string `json:"api" protobuf:"bytes,2,opt,name=api"`
 	// Credentials for Basic auth
 	BasicAuth *BasicAuthBitbucketServer `json:"basicAuth,omitempty" protobuf:"bytes,3,opt,name=basicAuth"`
+	// Credentials for AccessToken (Bearer auth)
+	BearerToken *BearerTokenBitbucket `json:"bearerToken,omitempty" protobuf:"bytes,5,opt,name=bearerToken"`
```

Comment:
To help with future contributions, I think is easier if the objects are added in the same order defined by the protobuf position.

Reply:
True, resolved!